src/views/Pipeline.vue

```diff
@@ -570,13 +576,6 @@ export default defineComponent({
     // state does not gets updated and hence the job configuration component takes it space in DOM
     await this.store.dispatch('job/updateCurrentJob', { job: {} });
   },
-  mounted(){
-    emitter.on("pinnedJobsUpdated", (this as any).updateSelectedPinnedJob);
-  },
-  unmounted(){
-    emitter.off('jobUpdated', this.updateJobs);
```

Comment:
We don't need to remove this jobUpdated emitter.

Reply:
ok sir I'll fix this

Reply:
Fixed sir
please have a look